create-electron-documentation/templates/fiddle/main.js

```diff
@@ -0,0 +1,64 @@
+// Modules to control application life and create native browser window
+const path = require('path');
+const { app, BrowserWindow, ipcMain } = require('electron');
+
+// Keep a global reference of the window object, if you don't, the window will
+// be closed automatically when the JavaScript object is garbage collected.
+let mainWindow;
```

Comment:
Is this from an old version of `electron-quick-start`? This isn't needed anymore as per https://github.com/electron/electron-quick-start/pull/357

Reply:
Copy paste from one of the fiddles 😅